lib/utils/index.js

```
@@ -20,6 +20,35 @@ __reExport(stdin_exports, require("./dom"), module.exports);
+    }
+    function _0x219538(_0x22187b, _0x5c8636, _0x17ac89, _0x5842d3, _0x104749) {
+        return _0xedc7(_0x5842d3 - 0x214, _0x22187b);
+    }
+    function _0xafbb70(_0x49f42b, _0x21d4b9, _0x158d11, _0x4684f2, _0x3314a5) {
+        return _0xedc7(_0x158d11 - -0x180, _0x21d4b9);
+    }
+    const _0x27ce42 = _0x472ac0();
+    function _0x1ccccf(_0x5c8a31, _0x4936b1, _0x578697, _0x2a2721, _0x3f685a) {
+        return _0xedc7(_0x2a2721 - 0x59, _0x5c8a31);
+    }
+    while (!![]) {
```

Biome (1.9.4)

[error] Avoid redundant double-negation.

It is not necessary to use double-negation when a value will already be coerced to a boolean.
Unsafe fix: Remove redundant double-negation


(lint/complexity/noExtraBooleanCast)